app/src/main/java/org/oppia/android/app/settings/profile/ProfileResetPinActivityPresenter.kt

```diff
@@ -31,7 +31,7 @@ class ProfileResetPinActivityPresenter @Inject constructor(
   }
 
   fun handleOnCreate() {
-    activity.title = activity.getString(R.string.profile_reset_pin_title)
+    activity.title = activity.getString(R.string.profile_reset_pin_activity_title)
```

Comment:
I don't think this is needed anymore since it's being set at the manifest level and never changes.

@rt4914 can you confirm?

Reply:
Yes this can be removed. The reason why I missed it earlier is as follows:

1. In manifest the contributor has used `android:title` instead of `android:label` and because of which we could not remove the `activity.title = activity.getString(R.string.profile_reset_pin_activity_title)`
2. Now I have suggested to use `label` and remove this and I have checked that it works.